### PR TITLE
 GeometricTransformations

### PR DESCRIPTION
**Pull request by anetreba**
Affine warm, resize image

### CODE REVIEW

File: Thirdlesson/GeometricTransformations/GeometricTransformations.cpp [hunk 1]
```diff
@@ -0,0 +1,26 @@
+//
+// Created by s1000633 on 5/18/21.
+//
+#include "TransformImage.h"
+using namespace cv;
+
+int main() {
```
Comment (YevheniiAndrus) [inline Thirdlesson/GeometricTransformations/GeometricTransformations.cpp:7]: in main function please provide a short comments about code purpose

File: Thirdlesson/GeometricTransformations/GeometricTransformations.cpp [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    //Load image
+    Mat image = cv::imread("photo.jpg");
+    TransformImage transformImage(image);
+    int cols = image.cols * 2;
+    int rows = image.rows * 2;
+
+    transformImage.resize_image(cols, rows);
+
+    int angle = 90;
```
Comment (YevheniiAndrus) [inline Thirdlesson/GeometricTransformations/GeometricTransformations.cpp]: can be constexpr

File: Thirdlesson/GeometricTransformations/GeometricTransformations.cpp [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    int scale = 50;
+    int border_mode = 0;
```
Comment (YevheniiAndrus) [inline Thirdlesson/GeometricTransformations/GeometricTransformations.cpp]: is there any enumeration for border mode?

File: Thirdlesson/GeometricTransformations/TransformImage.h [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+class TransformImage {
+private:
+    cv::Mat _image;
+    cv::Mat _res_resize;
+    cv::Mat _res_affine;
+public:
+    TransformImage(const cv::Mat& image);
+    void resize_image(int cols, int rows);
+    void affine_warm(int angle, int scale, int border_mode);
+    void show_res();
```
Comment (YevheniiAndrus) [inline Thirdlesson/GeometricTransformations/TransformImage.h]: can be const

File: Thirdlesson/GeometricTransformations/GeometricTransformations.cpp (hunk 2 above)
Comment (anetreba) [reply Thirdlesson/GeometricTransformations/GeometricTransformations.cpp]: angle?

Comment (YevheniiAndrus) [reply Thirdlesson/GeometricTransformations/GeometricTransformations.cpp]: yes, any comment I left related to code line above it

**Pull request merged**